product-tagger-template/web/routes/index.jsx

```diff
@@ -1,156 +1,28 @@
-  const {
-    submit,
-    control,
-    reset,
-    error: createError,
-    formState,
-  } = useActionForm(api.allowedTag.create);
-  // the useAction hook is used for deleting existing tags
-  const [{ error: deleteTagError }, deleteTag] = useAction(
-    api.allowedTag.delete
-  );
-
-  const removeTag = useCallback(
-    async (id) => {
-      // set the id of the deleted tag for disabling in UI
-      setDeletedTagId(id);
-      // call the deleteTag function defined with the useAction hook with the id of the tag to delete
-      await deleteTag({ id });
-    },
-    [deleteTag]
-  );
-
-  // render the page, using data, fetching, and error from the useFindMany, useAction, and useActionForm hooks to display different widgets
+export default function () {
+  // use autocomponents to automatically create a form and table to manage allowedTag records
   return (
     <Page title="Keyword manager">
       <Layout>
         <Layout.Section>
-          <TitleBar title="Manage keywords" />
-          <Form
-            onSubmit={async () => {
-              // submit the form and save the keyword as a new allowedTag record
-              await submit();
-              // reset the form input to empty
-              reset();
-            }}
-          >
-            <FormLayout>
-              {createError && (
-                <ErrorBanner title="Error adding keyword" error={createError} />
-              )}
-              <Controller
-                name="keyword"
-                control={control}
-                required
-                render={({ field }) => {
-                  // Functional components like the Polaris TextField do not allow for 'ref's to be passed in
-                  // Remove it from the props passed to the TextField
-                  const { ref, ...fieldProps } = field;
-                  // Pass the field props down to the TextField to set the value value and add onChange handlers
-                  return (
-                    <TextField
-                      label="Tag"
-                      type="text"
-                      autoComplete="tag"
-                      helpText={<span>Add a keyword</span>}
-                      disabled={formState.isSubmitting}
-                      connectedRight={
-                        <Button
-                          variant="primary"
-                          submit
-                          loading={formState.isSubmitting}
-                        >
-                          Add keyword
-                        </Button>
-                      }
-                      {...fieldProps}
-                    />
-                  );
-                }}
-              ></Controller>
-            </FormLayout>
-          </Form>
+          <Card>
+            {/** AutoForm automatically calls allowedTag.create on form submission */}
+            <AutoForm action={api.allowedTag.create} title="Add keywords" />
```

Comment:
I'm going to be adding creation success and error toasts for these in my next PR

Reply:
Is this to follow Shopify's Polaris guidelines? We already display feedback as a banner